### PR TITLE
Make All Elements Clickable

### PR DESCRIPTION
**Pull request by fcescob**
This PR addresses #1294

### CODE REVIEW

**Comment by kdembler** [comment]
Please add the atlas label in the future or request a review, this has been open for a week and I just discovered it accidentally

**Comment by kdembler** [comment]
For the review itself: the approach you've taken will require us to add click handlers for all the places where we use respective previews. This will make us duplicate the logic and make the potential future refactor more complicated.

What do you think about adding some Atlas-specific wrapper elements for `*Preview` components that would contain the navigation logic? Then all the views wouldn't need to worry about navigating but just use these components.

File: atlas/src/components/VideoGrid.tsx [hunk 1]
```diff
@@ -18,10 +19,13 @@ const VideoGrid: React.FC<VideoGridProps> = ({ videos }) => {
     <Grid>
       {videos.map((v, idx) => (
         <StyledVideoPreview
+          id={v.id}
+          channelId={v.channel.id}
           key={idx}
           title={v.title}
           channelName={v.channel.handle}
           channelAvatarURL={v.channel.avatarPhotoURL}
+          onChannelClick={() => navigate(routes.channel(v.channel.id))}
```
Comment (kdembler) [inline atlas/src/components/VideoGrid.tsx]: Do we still need that?

File: atlas/src/shared/components/InfiniteVideoGrid/InfiniteVideoGrid.tsx [hunk 2]
```diff
@@ -15,6 +17,7 @@ type InfiniteVideoGridProps = {
   ready?: boolean
   className?: string
 }
+console.log(Grid)
```
Comment (kdembler) [inline atlas/src/shared/components/InfiniteVideoGrid/InfiniteVideoGrid.tsx]: 😛

File: atlas/src/shared/components/InfiniteVideoGrid/InfiniteVideoGrid.tsx [hunk 3]
```diff
@@ -156,5 +161,8 @@ const StyledVideoPreviewBase = styled(VideoPreviewBase)`
   margin: 0 auto;
   width: 100%;
 `
+const StyledGrid = styled(Grid)`
+  padding-top: ${sizes.b2}px;
```
Comment (kdembler) [inline atlas/src/shared/components/InfiniteVideoGrid/InfiniteVideoGrid.tsx]: Why this change? In the design there is a 16px margin between title and the grid, like it is right now

File: atlas/src/components/VideoGrid.tsx (hunk 1 above)
Comment (fcescob) [reply atlas/src/components/VideoGrid.tsx]: Nice catch, removed.

File: atlas/src/shared/components/InfiniteVideoGrid/InfiniteVideoGrid.tsx (hunk 2 above)
Comment (fcescob) [reply atlas/src/shared/components/InfiniteVideoGrid/InfiniteVideoGrid.tsx]: Removed :D

**Review by kdembler — APPROVED** [review]
LGTM!

**Review by bedeho — APPROVED** [review]
